### DISCUSSION TITLE
ArcBox 3.0 community ideas backlog

### DISCUSSION THREAD

**Discussion by liorkamrat** [microsoft/azure_arc#1979] [Ideas]
Hi everyone,

Did you know that it's been 18 months since we announced ArcBox 2.0?

ArcBox deployments skyrocket 🚀 and we see hundreds of deployments per month, it's practically like donuts 🍩

So, we want to make it better and we have a lot of ideas in our backlog BUT it is also CRITICAL for us to collect the community feedback so now it's your chance. Our goal is to release ArcBox 3.0 in Q1 of 2024 and we plan the coding 👩‍💻 to start at the end of the summer ☀️. 

Share with us your thoughts, we would love to engage! 🙏

**Comment by rfsonders** [comment]
It would be good to find a way to lower the cost of a base ArcBox deployment. Or, time box. Or both. I have many clients that want to test and demo via ArcBox, but the footprint for consumption cost drive them away.

**Reply by likamrat** [reply]
Thank you @rfsonders, It's definitely top of mind for us. Here are some of the things we are thinking about:
- Deprecate the Cluster API (CAPI) method and go with something way smaller. This is applicable for ArcBox Full, DevOps, and DataOps.
- Have a deployment flag and an option for how many Arc-enabled servers you want. Based on the user's answer, we can decide on the size of the Azure VM hosting those nested VMs.
- We are also looking at an "ArcBox Local" option for next year as an option. The goal is for you to deploy ArcBox in a "Bring your own VM/Workstation/Laptop" fashion.

**Reply by duongthaiha** [reply]
Can I vote for the Bring you own devices please? My customer also have already invested in hardware on site and they want to validate Arc in there set up instead of a VM equivalent please.
